tests/wpt/web-platform-tests/domparsing/XMLSerializer-serializeToString.html

```diff
@@ -31,6 +31,13 @@ <h1>domparsing_XMLSerializer_serializeToString</h1>
   assert_equals(serialize(root), '<root><child1>value1</child1></root>');
 }, 'check XMLSerializer.serializeToString method could parsing xmldoc to string');
 
+test(function() {
+  var root = parse('<html><head></head><body><div></div><span></span></body></html>');
+  root.lastElementChild.append(document.createElement('div'));
+  root.lastElementChild.append(document.createElement('span'));
+  assert_equals(serialize(root), '<html><head/><body><div/><span/><div xmlns="http://www.w3.org/1999/xhtml"></div><span xmlns="http://www.w3.org/1999/xhtml"></span></body></html>');
```

Comment:
What is the output when you run this test in Servo? The serialized output contains two divs and two spans, and they have inconsistent xmlns attributes, so I am very surprised if this test passes.

Reply:
Oh, I skipped over the `root.lastElementChild.append` calls, which would explain the difference I guess.

Reply:
I think this test would be more clear without modifying the parsed document at all.

Reply:
only parse and then serialize?

Reply:
Exactly.